### ISSUE TITLE
Sound issues in EVERY game [PPSSPP 1.6.3] [PC]

### ISSUE THREAD

**Issue by ReilerTheDragonknight** [hrydgard/ppsspp#11470]
In every game that I play, the game itself runs normally, but the music/sound/audio runs way too fast. Even at 100% speed (which sometimes jumps to 99.6% or 100.6%), the game's audio is scratchy as hell, and plays too fast. I have tested this with the following games, and with various settings enabled and disabled, even trying to use the interpreter. (even homebrew games):
Monster Hunter (Freedom Unite and Portable 3rd HD): Game runs at 100%. Audio is scratchy to high hell and a lot of the music plays a tiny bit faster.
Rengoku II: Main menu music is REALLY fast, sounds distorted as hell.
Any Armored Core game: 3p and SLp sound awful, never did before. In Last Raven, it goes by fast, and even the game itself is sped up.
Kingdom Hearts: Birth By Sleep -Final Mix-: 60FPS or 30FPS, it sounds the exact same.
Basically every game I tested thus far: Choppy, fast music or audio that doesn't sync up with the game.

Just an FYI, I overclocked my computer, and I usually never had an issue like this.

Previously, before this, the games worked completely fine with zero issues, even outperforming my expectations at times. Not sure what went wrong.

I am running PPSSPP 1.6.3 on a humble potato Windows 7 PC. Not much else I can really say.

**Comment by LunaMoo** [comment]
I would guess you're using a timer hack or some other hackish setting like psp cpu over/under clocking to work around having a potato pc that can't run those games at full speed otherwise in which case restoring default settings should fix the problem, but if you really had to abuse some awful speedhacks, you can probably forget about perfectly playable psp emulation on your device.

**Comment by ReilerTheDragonknight** [comment]
I've tried with both my normal settings, of which I never had problems with, and the defaults, with the only thing being changed is the Anisotropic Filtering. Both just suddenly got bad.

**Comment by unknownbrackets** [comment]
It might also help to try changing the audio backend between "WASAPI" and "DSound", if you recently changed anything about your audio setup or audio drivers.  I think you might need to manually restart PPSSPP after changing the setting.

One fix was made in the latest git for an obscure audio format some audio devices (e.g. the headphone jack on a DS4 controller) use:
https://buildbot.orphis.net/ppsspp/

-[Unknown]

**Comment by ReilerTheDragonknight** [comment]
I tried that aswell. Doesn't help that the Audio tab in the options doesn't actually have many options, and the 'sound speedhack' is really vague. Switching between WASAPI and DSound doesn't work either. Trust me, I tried everything.

**Comment by unknownbrackets** [comment]
You said, "before this" it was working.  Did you change anything at all about your computer, OS, drivers, headset/speakers, etc.?  Was it fine before you overclocked?

Sound has been working fine for me on Windows 7, Windows Server 2012 (basically 8.1), and Windows 10.  Each with different audio devices and drivers.  So the trick will be figuring out what's different for you that's causing these issues.

-[Unknown]

**Comment by ReilerTheDragonknight** [comment]
Nope, didn't change anything. Drivers were the latest before and after OC, headset is still lasting me a good 2 years, the computer has always been the same. I use Realtek, and yes, it's also the latest version, and that never caused me any issues. Even reverting back didn't help with the problem.

**Comment by unknownbrackets** [comment]
Can you try another emulator, like Dolphin or PCSX2 or something?  I assume other apps (like music players and web browsers) play everything fine.

Under File -> Record it's possible to record a video.  First, select "Record Audio", and then after a little while select "Record Audio" again to stop it.

Under memstick/PSP/AUDIO you should see a wav file.  Does the audio sound distorted when you play this too, or is it fine here and only broken within the emulator?

I know you noted your PC specs as a potato, but the specific kind of potato may help.  For example, Jersey Royals or perhaps Charlotte?  But seriously, it can matter for example if you use a Core 2 Duo or a Core Solo or Celeron or an i7-8086K, or perhaps an AMD Athlon XP.

If you don't know, go to Settings -> Tools -> System information, and screenshot that.  Then go to CPU extensions and screenshot that too.

-[Unknown]

**Comment by ReilerTheDragonknight** [comment]
Can't exactly run those emulators. However, PS1, GBA, and SNES work completely fine. My PC specs are as follow: AMD Athlon II 170u, GeForce 6150SE nForce, 2 GB of RAM. Can't get new parts as I lack the money to do so.
When I both recorded with PPSSPP's internal recorder and Fraps... The PPSSPP record function sounded fine, with zero issues whatsoever. In Fraps, it picked up the audio issue.
The CPU extensions are:
SSE, SSE2, SSE3, SSE4A, and 64-bit support.

**Comment by LunaMoo** [comment]
Ancient cpu combined with just as ancient integrated graphics, it should not be capable of running those games at full speed outside of maybe menus or videos, hence why I still would suspect some unreliable speedhacks in use. Try default settings instead of your "normal settings", things like faster music and 100% speed can be caused by ini-only ppsspp setting to limit max fps for example and there are other hacks which are just as awful.

**Comment by unknownbrackets** [comment]
Could you say what you used to overclock?  Might help someone else who runs into this same problem.

Glad you got it working again.

-[Unknown]

**Comment by ReilerTheDragonknight** [comment]
Because I have an nForce motherboard, I used nTune. Specifically, I used it to have it automatically tune my system to the best settings, using the 3 hour tuner. It upgraded the mhZ from 200 to 230, and the PCIe Bus to 2925 Mhz. Here's what it looks like 
![image](https://user-images.githubusercontent.com/22803374/46920168-33000d00-cfe2-11e8-96d8-2cdf4e2a6008.png)
![image](https://user-images.githubusercontent.com/22803374/46920173-457a4680-cfe2-11e8-8352-87c4cbf94fd0.png)
Hopefully, my own dumb mistakes can help someone else, and maybe it'll help on finding a way to clamp the game speed despite the overclocking.

**Comment by unknownbrackets** [comment]
I wonder if the audio hardware wasn't able to keep up with the increased bus speed or something.  If you still want to try overclocking, I'd suggest trying individual parameters to see which one causes the audio to be off.

Though I don't personally overclock (I'd much prefer a stable system to worrying a crash or bug is due to overclock), I wouldn't call it dumb, it's not obvious this would happen or anything.

Thanks for posting this for others.

-[Unknown]
